### DISCUSSION TITLE
Organize multiple matches at the same time

### DISCUSSION THREAD

**Discussion by enricor91** [iamgiolaga/partitelle-bot#28] [Ideas]
A nice feature would be to be able to organize multiple matches at the same time in differents days.

**Comment by iamgiolaga** [comment]
Hey @rattino91, thanks for the idea! And sorry for the late reply 😔

At the moment, the whole behaviour relies on the assumption that you can have at most 1 active match per group.
Thanks to that, we store the latest message id to pin, and this is also needed to edit the message when new updates arrive, such as "Aggiungimi", "Proponimi" or "Toglimi".

Having multiple matches, means finding a way to target which match the single player refers to:

in other words, it's not enough that a player says "Aggiungimi" but it should say "Aggiungimi al match 1", and I think the UX would become less smooth.

In general, I like your proposal, but the whole system should be re-thought and, at the moment, I don't see quick solutions :/
